lib/resty/rediscluster.lua

```diff
@@ -159,24 +181,31 @@ local function try_hosts_slots(self, serv_list)
                 local servers = { serv_list = {} }
                 for n = 1, #slots_info do
                     local sub_info = slots_info[n]
-                    --slot info item 1 and 2 are the subrange start end slots
+                    -- slot info item 1 and 2 are the subrange start end slots
```

Comment:
I'd also recommend to completely break up this block into smaller more structured and understandable parts.

Reply:
Done with samples and code refactor.